akka-remote/src/main/scala/akka/remote/ByteStringUtils.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.remote
+
+import akka.annotation.InternalApi
+import akka.protobufv3.internal.UnsafeByteOperations
+import akka.util.ByteString
+import akka.protobufv3.internal.{ ByteString => ProtoByteString }
+import akka.util.ByteString.ByteString1C
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+private[akka] object ByteStringUtils {
+  def toProtoByteStringUnsafe(bytes: ByteString): ProtoByteString = {
+    if (bytes.isEmpty)
+      ProtoByteString.EMPTY
+    else if (bytes.isInstanceOf[ByteString1C]) {
```

Comment:
should we also include `|| (bytes.isInstanceOf[ByteString1] && bytes.isCompact)` in this case?